sdk/cosmos/azure-cosmos-encryption/README.md

```diff
@@ -44,13 +201,16 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
 <!-- LINKS -->
-[source_code]: src
+[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/cosmos/azure-cosmos/src
```

Comment:
nit - we don't need this link.

Reply:
done , removed other non used url as well